src/App.js

```diff
@@ -3,18 +3,51 @@ import React from "react";
 
 const IMAGE_HEIGHT = 200;
 
-class Image extends React.Component {
+class ImageList extends React.Component {
+
+    render() {
+        const cats = this.props.cats;
+        return (
+            <div>
+                {cats}
+            </div>
+        );
+    }
+}
+
+
+class InputForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(e) {
+        this.props.onCountChange(e.target.value);
+    }
+
+    render() {
+        const count = this.props.count;
+        return (
+            <fieldset>
+                <legend>Enter count</legend>
```

Comment:
Minor: in a real project, it would be `<label>`. We use `<legend>` when we need to label several fields.